Scroller/EditableWrapper.js

```diff
@@ -36,6 +39,9 @@ const EditableWrapper = (props) => {
 		fromIndex: null,
 		prevToIndex: null,
 
+		// Temporal stored value
```

Comment:
It looks better to write here when `nextSpotlightRect` is used to avoid misunderdtanding since the name of the variable is so general.

Reply:
Thanks. I fixed.